src/main/java/com/eppo/sdk/dto/EppoAttributes.java

```diff
@@ -15,4 +19,62 @@ public EppoAttributes() {
   public EppoAttributes(Map<String, EppoValue> initialValues) {
     super(initialValues);
   }
+
+  public String serializeToJSONString() {
+    return EppoAttributes.serializeAttributesToJSONString(this);
+  }
+
+  public static String serializeAttributesToJSONString(Map<String, ?> attributes) {
+    return EppoAttributes.serializeAttributesToJSONString(attributes, false);
+  }
+
+  public static String serializeNonNullAttributesToJSONString(Map<String, ?> attributes) {
+    return EppoAttributes.serializeAttributesToJSONString(attributes, true);
+  }
```

Comment:
Unsure yet if users will want to include or omit attributes with `null` values, so providing both options. (Our proxy omits them)